### PR TITLE
Move fonts to CDN

### PR DESCRIPTION
**Pull request by naveed-ahmad**
### Summary
TSIA

### CODE REVIEW

File: src/components/Fonts/FontPreLoader.tsx [hunk 1]
```diff
@@ -2,17 +2,15 @@ import React from 'react';
...
 const LOCALE_PRELOADED_FONTS = {
-  [DEFAULT_LOCALE]: [
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_regular.woff2' },
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_semibold.woff2' },
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_bold.woff2' },
-  ],
-  ar: [{ type: 'font/woff2', location: 'urdu/NotoNaskhArabic-Regular.woff' }],
-  bn: [{ type: 'font/ttf', location: 'NotoSerifBengali/NotoSerifBengali-Regular.ttf' }],
-  ur: [{ type: 'font/woff2', location: 'urdu/NotoNaskhArabic-Regular.woff' }],
+  [DEFAULT_LOCALE]: [{ type: 'font/woff2', location: 'lang/ProximaVara/ProximaVara.woff2' }],
```
Comment (osamasayed) [inline src/components/Fonts/FontPreLoader.tsx:10]: We were using proxima_vara_semibold and proxima_vara_bold intentionally for non-default `font-weight` values.

Comment (osamasayed) [inline src/components/Fonts/FontPreLoader.tsx:10]: We were using `proxima_vara_semibold` and `proxima_vara_bold` intentionally for non-default `font-weight` values.

File: src/styles/global.scss [hunk 2]
```diff
@@ -14,17 +14,43 @@ body {
   }
 
   overscroll-behavior-y: none;
-  font-family: "ProximaVara", "Helvetica Neue", Helvetica, Arial;
+  font-family: "Helvetica Neue", Helvetica, Arial;
```
Comment (osamasayed) [inline src/styles/global.scss:17]: Why are we we removing `ProximaVara` from here here?

File: src/styles/fonts.scss [hunk 3]
```diff
@@ -1,96 +1,102 @@
...
+// Fallback for browsers that don't support variable fonts
 @font-face {
-  font-family: "NotoSerifBengali";
-  src: url("/fonts/NotoSerifBengali/NotoSerifBengali-Regular.ttf");
+  font-family: "ProximaVara";
+  src: local("Proxima Vara"),
+    url("#{$font-cdn}/lang/ProximaVara/ProximaVara.woff2");
+  font-weight: 100 900;
+  /* default 400 */
+  font-stretch: 50% 100%;
```
Comment (osamasayed) [inline src/styles/fonts.scss:99]: Why do we need this?

**Comment by osamasayed** [comment]
@naveed-ahmad I suggest we remove the font files from the repo since we are not longer using them.

**Comment by naveed-ahmad** [comment]
@osamasayed https://github.com/quran/quran.com-frontend-next/pull/1139

File: src/components/Fonts/FontPreLoader.tsx (hunk 1 above)
Comment (osamasayed) [reply src/components/Fonts/FontPreLoader.tsx:10]: It seems the new font being used is less in size that the 3 files we used to use collectively:

Before:
Screen Shot 2022-01-25 at 09 39 32

After:
Screen Shot 2022-01-25 at 09 39 27

**Review by osamasayed — APPROVED** [review]

**Pull request merged**